python_modules/dagster/dagster/_core/definitions/decorators/decorator_assets_definition_builder.py

```diff
@@ -429,9 +429,9 @@ def from_asset_outs_in_asset_centric_decorator(
         keys_by_output_name = make_keys_by_output_name(named_outs_by_asset_key)
         internal_deps = {keys_by_output_name[name]: asset_deps[name] for name in asset_deps}
 
-        _validate_check_specs_target_relevant_asset_keys(
-            passed_args.check_specs, list(named_outs_by_asset_key.keys())
-        )
+        # _validate_check_specs_target_relevant_asset_keys(
```

Comment:
I think this is essential, unfortunately. The idea is that we explicitly only want to be targeting asset keys that produce materializations on the downstream multi asset.